### PR TITLE
Add replacement_enum to font_$_id and logo_$_id

### PR DESCRIPTION
**Pull request by reiterl**
Use it in a check in set_font and set_logo.
Add some tests.

### CODE REVIEW

File: openslides_backend/action/actions/meeting/base_set_mediafile_action.py [hunk 1]
```diff
@@ -49,7 +50,18 @@ def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
...
+        place_field = Meeting().get_field(self.field)
+        allowed_places = (
+            place_field.replacement_enum
+            if isinstance(place_field, BaseTemplateField)
+            else None
+        )
+        if allowed_places and place not in allowed_places:
+            raise ActionException(
+                f"Invalid place: {place}, allowed are {allowed_places}"
+            )
```
Comment (jsangmeister) [inline openslides_backend/action/actions/meeting/base_set_mediafile_action.py:63]: I *think* the relation handling already does this check. Did you try it out without this implementation?

File: global/meta/models.yml [hunk 2]
```diff
@@ -1490,18 +1490,31 @@ meeting:
       to: mediafile/used_as_logo_$_in_meeting_id
       required: false
     restriction_mode: B
+    replacement_enum:
+      -  projector_main
```
Comment (jsangmeister) [inline global/meta/models.yml:1494]:
```suggestion
      - projector_main
```

**Review by jsangmeister — CHANGES_REQUESTED** [review]

File: global/meta/models.yml (hunk 2 above)
Comment (reiterl) [reply global/meta/models.yml]: Ah, a typo, fixed.

**Review by r-peschke — CHANGES_REQUESTED** [review]

File: tests/system/action/meeting/test_set_font.py [hunk 3]
```diff
@@ -31,11 +31,34 @@ def test_set_font_correct(self) -> None:
             }
         )
         response = self.request(
-            "meeting.set_font", {"id": 222, "mediafile_id": 17, "place": "web_header"}
+            "meeting.set_font", {"id": 222, "mediafile_id": 17, "place": "bold"}
         )
         self.assert_status_code(response, 200)
         model = self.get_model("meeting/222")
-        assert model.get("font_$web_header_id") == 17
+        assert model.get("font_$bold_id") == 17
```
[the remark this answers is not in the document]
Comment (reiterl) [reply tests/system/action/meeting/test_set_font.py]: Also assert the template field and use assert_model_exists.

File: tests/system/action/meeting/test_set_logo.py [hunk 4]
```diff
@@ -37,6 +37,29 @@ def test_set_logo_correct(self) -> None:
         model = self.get_model("meeting/222")
         assert model.get("logo_$web_header_id") == 17
```
[the remark this answers is not in the document]
Comment (reiterl) [reply tests/system/action/meeting/test_set_logo.py]: Assert the template field, too. And use assert_model_exists.

**Review by r-peschke — APPROVED** [review]

**Pull request merged**